contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/secure-boot/jobs.pxu

```diff
@@ -0,0 +1,28 @@
+plugin: manual
+category_id: secure-boot
```

Comment:
we could use the security category in base provider. It seems the same testing purpose.
https://github.com/canonical/checkbox/blob/main/providers/base/units/security/category.pxu

Reply:
Roger that!